src/main/java/com/google/cloud/tools/gradle/endpoints/framework/server/task/GenerateClientLibsTask.java

```diff
@@ -81,8 +81,7 @@ public void setServiceClasses(List<String> serviceClasses) {
   @TaskAction
   void generateClientLibs() throws Exception {
 
-    getProject().delete(clientLibDir);
-    clientLibDir.mkdirs();
+    // This action intentionally does not clean the output directory
```

Comment:
Also, this task isn't particularly part of any chain in the plugin, it doesn't really affect other processes.